InputValidation/IValidation.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+/// <summary>
+/// Interface for various forms of input validation in TotalImage.
+/// </summary>
+namespace TotalImage.Validation
+{
+    public interface IValidation
```

Comment:
Could this be a generic interface - ie IValidation<T> to preserve type information on Validate?